Source/WebCore/platform/graphics/avfoundation/objc/LocalSampleBufferDisplayLayer.h

```diff
@@ -98,6 +98,7 @@ class WEBCORE_EXPORT LocalSampleBufferDisplayLayer final : public SampleBufferDi
 #endif
 
     WorkQueue& workQueue() { return m_processingQueue.get(); }
+    Ref<WorkQueue> protectedWorkQueue() { return m_processingQueue.get(); }
```

Comment:
Would we mark `m_processingQueue` as `const` instead?

Reply:
Good point. Fixed.